frontend/src/Components/AppTitle/AppTitle.tsx

```diff
@@ -0,0 +1,26 @@
+import type { ReactNode } from 'react';
+import styled from 'styled-components';
+
+type AppTitleProps = {
+  children: ReactNode;
+};
+
+export default function AppTitle({ children }: AppTitleProps) {
+  return <Apptitle>{children}</Apptitle>;
+}
+
+const Apptitle = styled.h1`
+  text-align: center;
+  justify-self: center;
+  width: 100%;
+  margin: 0;
+  font-size: 3rem;
+  font-weight: normal;
+  margin-bottom: 0.5rem;
+  padding-top: 4px;
+  padding-bottom: 4px;
+  color: var(--color-primary);
+  background-color: var(--color-background-primary);
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  border-radius: 0px 0px 5px 5px;
+`;
```

Comment:
Seems like a lot of CSS for a single Heading

Reply:
it seems so, but looks very good :)